### ISSUE TITLE
Inclure la version d'Omega dans la barre de titre

### ISSUE THREAD

**Issue by cent20net** [Omega-Numworks/Omega#133]
C'est très bête, mais inclure la version d'Omega dans la barre de titre permet aux élèves de se rendre compte qu'ils n'ont pas la même version que leur camarade, et le prof qui rencontrerait éventuellement un pb peut dire : Il te suffit d'installer la nouvelle version de Omega !

**Comment by cent20net** [comment]
I have to speak English ?

**Comment by quentinguidee** [comment]
Yes, since we have contributors that speaks English; but don't worry I translate them 😉

**Comment by M4xi1m3** [comment]
I kindof understand why this is an issue in the first place but I also don't like the way of solving it. Having version in toolbar makes it loaded with yet something more. Maybe if the version would be written smaller? But then it would be inconsistent. I really don't know tbh.
